Docs: Feedback for 'Self-Contained docToolchain'

I tried following the documentation on how to get a self-contained version of doctoolchain (without docker). Sadly this did not work out as expected.

## Environment
- Windows 10
- CMD
- no previous installation of Gradle present (Maven user ;-) )
- unrestricted network/internet access

## Steps executed
- http://doctoolchain.org/docToolchain/v2.0.x/020_tutorial/010_Install.html --> "Get Wrapper" --> Worked
- `dtcw.bat tasks --group=doctoolchain` --> to test install --> Worked (Build Successful) --> but no task for `getJava` available
- `dtcw.bat local getJava` --> **BUILD FAILURE**

## Problem encountered
```commandline
FAILURE: Build failed with an exception.

* What went wrong:
Task 'getJava' not found in root project 'docToolchain'.
```

## Question/Issue
Just following the docs I did not find a solution for this issue. 

---
#page: https://github.com/doctoolchain/doctoolchain/edit/ng/src/docs/020_tutorial/120_self-contained-dtc.adoc
#branch: ng

--- a/changelog.adoc
+++ b/changelog.adoc
@@ -16,6 +16,7 @@ and this project tries to adhere to https://semver.org/spec/v2.0.0.html[Semantic
 * https://github.com/docToolchain/docToolchain/pull/951[#951 Improve DTC_PROJECT_BRANCH management]
 * https://github.com/docToolchain/docToolchain/issues/976[#976 dtcw shows a fatal error when not in git repository]
 * https://github.com/docToolchain/docToolchain/issues/978[#978 dtcw.ps1 does not correctly check for java version]
+* https://github.com/docToolchain/docToolchain/issues/890[#890 updated documentation on "Self-Contained docToolchain"]
 
 === added
 

--- a/src/docs/020_tutorial/120_self-contained-dtc.adoc
+++ b/src/docs/020_tutorial/120_self-contained-dtc.adoc
@@ -17,6 +17,7 @@ In order to fetch the docker image from your local docker proxy, please check th
 
 If you don't have Docker available, you need a local, self-contained install.
 
+==== Download wrapper
 Start on an unrestricted system and download the docToolchain wrapper:
 
 [source, bash]
@@ -27,7 +28,8 @@ curl -Lo dtcw doctoolchain.github.io/dtcw
 chmod +x dtcw
 ----
 
-We start wit ensuring that we have the correct Java Development Kit (JDK) installed.
+==== Check and install correct Java Development Kit (JDK)
+We start with ensuring that we have the correct Java Development Kit (JDK) installed.
 Execute
 
 [source, bash]
@@ -38,6 +40,7 @@ Execute
 to fetch and install the correct JDK.
 It will be installed to `$HOME/.doctoolchain/jdk`.
 
+==== Start first tasks to install docToolchain locally
 Now, let's start a simple task to download and install docToolchain locally:
 
 [source, bash]
@@ -52,6 +55,7 @@ When you run this command, docToolchain will ask if it is allowed to install its
 It will also fetch some first dependencies in order to execute the task.
 These dependencies will be downloaded to `$HOME/.doctoolchain/.gradle`.
 
+==== Fetch remaining dependencies
 To fetch the remaining dependencies, execute
 
 [source, bash]
@@ -61,8 +65,20 @@ To fetch the remaining dependencies, execute
 
 this will fetch the remaining dependencies and download them to `$HOME/.doctoolchain/.gradle`
 
+==== Get an offline gradle binary
+The last needed step is to download an offline version of `gradle` for docToolchain to use.  
+This was inspired by https://chengl.com/gradle-offline-build/.
 
-As a result, we now have docToolchain, a JDK and all dependencies installed to `$HOME/.doctoolchain`.
+.Steps
+. Download a `gradle` binary-zip from https://services.gradle.org/distributions/ 
+. Place it in the folder `.doctoolchain/docToolchain-$VERSION/gradle/wrapper` (replace $VERSION with current docToolchain version that you installed) 
+. Go to folder `.doctoolchain/docToolchain-$VERSION/gradle/wrapper` and open the file `gradle-wrapper.properties`
+.  change the `distributionUrl` to `distributionUrl=gradle-$GRADLEVERSION-bin.zip` (replace $GRADLEVERSION with the version of the binary-zip you just downloaded)
+
+docToolchain now uses an offline version of Gradle.
+
+==== Finalize
+As a result, we now have docToolchain, a JDK, gradle and all dependencies installed to `$HOME/.doctoolchain`.
 
 You can now zip this folder and copy it to your restricted environment to the `$HOME/.doctoolchain` folder.
 The wrapper of your project will recognize it as local installation and use it.

--- a/src/docs/10_about/30_community.adoc
+++ b/src/docs/10_about/30_community.adoc
@@ -73,5 +73,6 @@ Please get in touch to update your entry or let us know if you have contributed 
 - https://github.com/wonderbird[Stefan Boos]
 - https://github.com/adrpar[Adrian Partl]
 - https://github.com/bjkastel[Bjoern Kasteleiner]
+- https://github.com/SidekickJohn[Jan Küfner]
 
 image::https://img.shields.io/github/contributors/doctoolchain/doctoolchain.svg[link=https://github.com/docToolchain/docToolchain/graphs/contributors]

